### PR TITLE
WIP: Sync AnacondaRecipes

### PR DESCRIPTION
**Pull request by jakirkham**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @mingwandroid

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,18 +6,20 @@ package:
...
   sha256: 7f323f0e0c18488748f3d9b2d4353f00e904ea2ccd0372ea04d7f77aa1156557
   patches:
+    # - patches/0005-CMake-Fix-windres-.rc-not-including-programs-windres.patch
+    # Causes linking errors due to benchmarking not beging built into the shared library on
+    # Windows (or it may be there are some missing __declspec(dllexport) in the shared library?)
+    # - patches/0006-Link-zstd-exe-dynamically.patch
+
+build:
+  number: 0
     - patches/0001-Fix-VS2008-solution-format-version.patch          # [win]
```
Comment (xhochy) [inline recipe/meta.yaml]: This diff looks really broken as the patches were removed in master.

**Comment by jakirkham** [comment]
Yeah it’s also pretty confused especially since both branches use CMake, but it makes it look like that is not the case.

**Comment by jakirkham** [comment]
FTR the diff only looks weird on GitHub. I'm not seeing these issues with Git.

**Comment by jakirkham** [comment]
> Can you rebase on master?

That's probably not easy. It's probably more straightforward to fix the merge conflicts. I'll try to work on this.

> The diff looks very weird and thus I feel unsafe with merging these changes.

Agreed. I don't think it should be merged until we sort out the merge conflicts. I'm hopeful GitHub will regain its sense then. 🙂

**Comment by wolfv** [comment]
Is this PR superseded?

**Comment by xhochy** [comment]
This is quite outdated again.

**Pull request closed without merging**